src/Directory.Build.props

```diff
@@ -1,23 +1,21 @@
 <Project>
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+	<Import
+		Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 	<PropertyGroup>
-		<Company>Microsoft Corporation</Company>
-		<Product>Microsoft Sql Tools</Product>
-	</PropertyGroup>
-	<PropertyGroup>
-		<Authors>Microsoft</Authors>
 		<EmbedUntrackedSources>true</EmbedUntrackedSources>
 		<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
 		<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-		<Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
 		<PackageLicenseExpression>MIT</PackageLicenseExpression>
 		<PackageIconUrl>http://s.gravatar.com/avatar/6f39d8562df0a3509a8240fb281bc5bd?s=80</PackageIconUrl>
 		<PackageProjectUrl>https://github.com/Microsoft/sqltoolsservice/</PackageProjectUrl>
 		<RepositoryUrl>https://github.com/Microsoft/sqltoolsservice</RepositoryUrl>
 		<RepositoryType>git</RepositoryType>
 		<PackageTags>SQL XPLAT</PackageTags>
-	</PropertyGroup>
-	<PropertyGroup>
 		<DebugType>portable</DebugType>
+		<!-- add Source Link info to package/pdbs-->
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
 	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.SourceLink.GitHub" />
```

Comment:
>	<PackageReference Include="Microsoft.SourceLink.GitHub" /> [](http://example.com/codeflow?start=1&length=59)

this could be a GlobalPackageReference in packages.props instead
If you leave it here you should set PrivateAssets=All at least.

Reply:
and do you need to pass `/p:ContinuousIntegrationBuild=true` in your official build to get the full benefit?

Reply:
Updated. Decided to go with property instead since we just have a single build script and I didn't feel like it was worth the effort to plumb through a whole new parameter just for this (and it also ensures we always have the property set in the pipelines)